packages/react-components/src/components/Popover/Popover.tsx

```diff
@@ -15,6 +17,7 @@ export interface IPopoverProps {
   className?: string;
   placement?: Placement;
   isVisible?: boolean;
+  flipOptions?: Partial<FlipOptions & DetectOverflowOptions>;
```

Comment:
Can we showcase this within `Storybook`?

Reply:
Consider this solution

```suggestion
  flipOptions?: Parameters<typeof flip>[0];
```

Reply:
The provided link doesn't show this (and some of the others) props available for this component.
Please fix that, because currently there is impossible to check how their props work to check the component behavior.

Reply:
@sgraczyk It works and if you think it's better that way, I can leave it but IMHO reaching the first parameter with "[0]" is quite cryptic.

Reply:
@MichalPaszowski It's easier to maintain compatibility with the library that way. It may look strange, but we're actually interested in the first (and only) parameter of a function.

Reply:
There may be a better way in TS-land but I'm not aware of it.